src/sentry/flags/providers.py

```diff
@@ -121,16 +141,43 @@ def handle_launchdarkly_event(
     ]
 
 
-def handle_provider_event(
-    provider: str,
-    request_data: dict[str, Any],
-    organization_id: int,
+"""Splitio Integration.
+
+Enabling:
+    - Go to Split.io page.
+    - Admin settings.
+    - Integrations.
+    - Outgoing webhooks.
+    - Select environments and paste url.
+
+Documentation: https://help.split.io/hc/en-us/articles/360020957991-Webhook-audit-log
+"""
+
+
+DELETED_REGEX = re.compile(r"^.* deleted feature .* with comment .*")
+
+
+def handle_splitio_event(
+    request_data: dict[str, Any], organization_id: int
 ) -> list[FlagAuditLogRow]:
-    match provider:
-        case "launchdarkly":
-            return handle_launchdarkly_event(request_data, organization_id)
-        case _:
-            raise InvalidProvider(provider)
+    if "previous" not in request_data:
+        action = ACTION_MAP["created"]
+    elif DELETED_REGEX.match(request_data["description"]):
+        action = ACTION_MAP["deleted"]
+    else:
+        action = ACTION_MAP["updated"]
+
+    return [
+        {
+            "action": action,
+            "created_at": timestamp_to_datetime(request_data["time"]),
+            "created_by": request_data["editor"],
+            "created_by_type": CREATED_BY_TYPE_MAP["name"],
+            "flag": request_data["name"],
+            "organization_id": organization_id,
+            "tags": {"environment": request_data["environmentName"]},
```

Comment:
should we standardize what we put in the tags? or does it not really matter?

i could add `environment` to the LD tags as well. looks like the splitio request body also has a description which i added to the LD tags intially

Reply:
A standard tag should just be a column.  Tags are meant for unstructured data that could be displayed to the user but not otherwise can not be part of the structured record.